src/renderer_mtl.h

```diff
@@ -390,10 +390,14 @@ namespace bgfx { namespace mtl
 
 		bool supportsTextureSampleCount(int sampleCount)
 		{
-			if (BX_ENABLED(BX_PLATFORM_VISIONOS) || (BX_ENABLED(BX_PLATFORM_IOS) && !iOSVersionEqualOrGreater("9.0.0")) )
+			if (BX_ENABLED(BX_PLATFORM_IOS) && !iOSVersionEqualOrGreater("9.0.0"))
```

Comment:
visionOS 1.0 supports `supportsTextureSampleCount:` so I wasn't sure why this was being checked here. Happy to revert if there was a reason for this

Reply:
This was probably a mistake on our side. If it's supports it - let's use it 👍🏻